sdk/keyvault/Azure.Security.Keyvault.Keys/src/RSAKey.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.Security.Keyvault.Keys
+{
+    public class RSAKey
+    {
+        public int KeySize { get; set; }
+        public string Name { get; set; }
+        public JsonWebKey KeyMaterial { get; set; }
```

Comment:
>public JsonWebKey KeyMaterial { get; set; } [](start = 8, length = 43)

If this class is intended as an input bag for CreateKey it shouldn't have KeyMaterial on it.  Only a handful of properties on the JWK are actually settable at creation time.